octobot_commons/singleton/singleton_class.py

```diff
@@ -17,3 +17,14 @@ def instance(cls, *args, **kwargs):
         if cls not in cls._instances:
             cls._instances[cls] = cls(*args, **kwargs)
         return cls._instances[cls]
+    
+    @classmethod
+    def get_only_instance(cls):
```

Comment:
I would have named it `get_instance_if_exists()`

Reply:
done